.changeset/afraid-lizards-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
+---
+
+Updated ErrorBoundary logging to include the componenetStack as well as the widget type as metadata.
```

Comment:
For trivial changes there is the option to run `yarn changeset --empty`, but I think it's safer to add a changeset for posterity. 

```suggestion
Updated ErrorBoundary logging to include the componentStack as well as the widget type as metadata.
```